sitemap.xml

```diff
@@ -31,7 +31,7 @@ category: hidden
   {% for post in site.posts %}
     {% unless post.published == false %}
       <url>
-        <loc>{{ site.url }}{{ post.url }}</loc>
+        <loc>{{ site.a_rootpage }}{{ post.url }}</loc>
```

Comment:
What does this do?

Reply:
That's from #988 - @gilgongo should know.

Reply:
@ann0see  That was to fix the fact that we weren't publishing fully-qualified paths for the KB URLs in sitemap.xml. Google complained they were invalid so wouldn't index them.

Reply:
Ok.